src/core/managers/NotificationsFeedManager/NotificationsFeedManager.ts

```diff
@@ -116,24 +118,19 @@ export class NotificationsFeedManager extends SidePanelBaseManager<
   }
 
   protected async updateDataAndNotifications() {
-    const {
-      transactions: sessions,
-      account,
-      toasts,
-      network
-    } = this.store.getState();
-
-    const { pendingTransactions } = createToastsFromTransactions({
+    const { transactions, account, toasts, network } = this.store.getState();
+
+    const { pendingTransactionToasts } = await createToastsFromTransactions({
       toastList: toasts,
-      sessions,
+      sessions: transactions,
```

Comment:
inside `createToastsFromTransactions`, why do we use the concept of `sessions` and not directly `transactions` ?
not mandatory comment